baked_in.go

```diff
@@ -2436,3 +2437,41 @@ func isDnsRFC1035LabelFormat(fl FieldLevel) bool {
 	val := fl.Field().String()
 	return dnsRegexRFC1035Label.MatchString(val)
 }
+
+// isCreditCard is the validation function for validating if the current field's value is a valid credit card number
+func isCreditCard(fl FieldLevel) bool {
+	val := fl.Field().String()
+	var creditCard bytes.Buffer
+	segments := strings.Split(val, " ")
+	for _, segment := range segments {
+		if len(segment) < 3 {
+			return false
+		}
+		creditCard.WriteString(segment)
+	}
+
+	ccDigits := strings.Split(creditCard.String(), "")
+	size := len(ccDigits)
+	if size < 12 || size > 19 {
```

Comment:
Hi,
I wrote this comment below the previous comments' thread. 
However just in case putting it also here

The correct PAN validation nowadays should be `size < 10 || size > 19`

Here is the official statement:

https://www.iso.org/news/2016/11/Ref2146.html

Within the current version of ISO/IEC 7812-1, an IIN is defined as a fixed-length numeric of six digits. The standard also defines the Primary Account Number (PAN), a number which is used to identify an individual account holder. The PAN is of variable length, ranging from 8 to 19 digits.

Due to the increasing number of card issuers, there is expected to be a shortage in the available supply of IINs. Therefore, ISO/IEC 7812-1 is being revised to expand the IIN to an eight-digit numeric value from the current six digits. The PAN will continue to remain a variable length, `ranging from 10 to 19 digits.`